ta/arch/arm/link.mk

```diff
@@ -5,6 +5,14 @@ link-script-dep$(sm) = $(link-out-dir$(sm))/.ta.ld.d
 SIGN_ENC ?= $(PYTHON3) $(ta-dev-kit-dir$(sm))/scripts/sign_encrypt.py
 TA_SIGN_KEY ?= $(ta-dev-kit-dir$(sm))/keys/default_ta.pem
 
+# In case the TA offline-signing has been performed.
+ifeq ($(CFG_STITCH_TA),y)
+# The signatures directory. The linking process expects <ta-uuid>.sig files in it.
+TA_SIG_DIR ?= signatures
+# The hash and signing algorithm.
+TA_SIGN_ALG ?= TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256
```

Comment:
I would prefer
```make
ifneq (,$(TA_SIGN_ALG))
crypt-args$(user-ta-uuid) += --algo $(TA_SIGN_ALG)
endif
```
...so that the default algorithm stays in `sign_encrypt.py` and is not duplicated here.

Reply:
Makes sense of cource, much better!

Reply:
And we could (should) even wait for a real use case before adding this variable. Do you plan on using a different signing scheme?

Reply:
Yes, our HSM server uses (at the moment) the PKCS1.5, so without this variable we wouldn't be able to stitch the signature. Id add this variable and remove it if/when the `sign_encrypt.py` script supports exactly one signature scheme. If however you think it is not necessary, I can add it locally just in my case.